scripts/build/termux_step_get_dependencies.sh

```diff
@@ -8,7 +8,16 @@ termux_step_get_dependencies() {
 		termux_download_repo_file
 	fi
 
-	while read PKG PKG_DIR; do
+	local command_arg="READ_ERROR"
+	local return_value=0
+	while read PKG PKG_DIR; return_value=$?; [[ $return_value -eq 0 ]]; do
```

Comment:
```suggestion
	while read PKG PKG_DIR; return_value=$?; (( return_value )); do
```

Reply:
```shell
$ return_value=0
$ (( return_value )) && echo 1

$ return_value=1
$ (( return_value )) && echo 1
1
```

And what does `(( return_value ))` mean by looking at it, booleans are also anything not equal to `0`. The `-eq 0` makes it clear easily, line length is irrelevant.